docs/src/man/categorical.md

```diff
@@ -80,7 +80,7 @@ julia> sort(cv)
 
 ```
 
-By default, a `CategoricalArray` is able to represent 2<sup>32</sup> different levels.
+By default, a `CategoricalArray` is able to represent ``2^32`` different levels.
```

Comment:
what do double backquotes mean?

Reply:
@nalimilan - bump - I would like to understand the change before merging :)

Reply:
That's the syntax for equations. It's too bad we need such an advanced feature for this, but I guess it's better than using Unicode superscripts?